src/server/helpers/html.js

```diff
@@ -3,6 +3,25 @@ import ReactDOM from 'react-dom/server';
 import Helmet from 'react-helmet';
 import { Provider } from 'react-redux';
 
+import { conf } from '../helpers/config';
+
+const GAID = conf.get('GOOGLE_ANALYTICS_ID');
+
+const googleTagManager = GAID ?
+  <script
+    dangerouslySetInnerHTML={{ __html: `(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer', '${GAID}')` }}
+  /> : null;
+
+const googleTagManagerNoScript = GAID ?
+  <noscript
+    dangerouslySetInnerHTML={{ __html: `<iframe src="https://www.googletagmanager.com/ns.html?id=${GAID}"
```

Comment:
Any reason for using `dangerouslySetInnerHTML` rather then just having `<iframe>` as child element?